### ISSUE TITLE
Is there a way to influence which direction (up/down) the dropdown appears?

### ISSUE THREAD

**Issue by pdaniel37** [zsoerenm/chips_input#19]
![image](https://user-images.githubusercontent.com/15150261/158879198-cfab26d2-0690-44fb-a4ae-a639d7418dca.png)

**Comment by zsoerenm** [comment]
No, unfortunately RawAutocomplete does not allow to customize the location of the overlay.
You might follow this issue here: https://github.com/flutter/flutter/issues/74509
